### PR TITLE
Implemented all requirements 

### PR DESCRIPTION
**Pull request by AlexandruBaris**

### CODE REVIEW

File: src/main/java/com/homework/homeworkWebApp/exceptions/AlreadyExists.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.homework.homeworkWebApp.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class AlreadyExists extends RuntimeException {
```
Comment (vermurachi) [inline src/main/java/com/homework/homeworkWebApp/exceptions/AlreadyExists.java]: Add suffix Exception

File: src/main/java/com/homework/homeworkWebApp/model/Employee.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package com.homework.homeworkWebApp.model;
+
+import lombok.*;
```
Comment (vermurachi) [inline src/main/java/com/homework/homeworkWebApp/model/Employee.java]: Remove wild card imports

File: src/main/java/com/homework/homeworkWebApp/service/implementations/EmployeeServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            employee.setPhoneNumber(employeeDto.getPhoneNumber());
+            employee.setSalary(employeeDto.getSalary());
+            repository.save(employee);
+        }
+        return EmployeeDto.from(employee);
+    }
+
+    private void verifyPhoneAndEmail(EmployeeDto employeeDto){
+//        if(repository.findEmployeeByEmail(employeeDto.getEmail()).isPresent()){
+//            throw new AlreadyExists("This email: " + employeeDto.getEmail() + " is already used");
```
Comment (vermurachi) [inline src/main/java/com/homework/homeworkWebApp/service/implementations/EmployeeServiceImpl.java]: Remove commented code

File: src/main/java/com/homework/homeworkWebApp/service/implementations/EmployeeServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+//        }
+//        if(repository.findEmployeeByPhoneNumber(employeeDto.getPhoneNumber()).isPresent()){
+//            throw new AlreadyExists("This phone number : " + employeeDto.getPhoneNumber() + " is already used");
+//        }
+        boolean phone, email;
+        phone = repository.findEmployeeByPhoneNumber(employeeDto.getPhoneNumber()).isPresent();
+        email = repository.findEmployeeByEmail(employeeDto.getEmail()).isPresent();
+        switch (phone+"-"+email){
+            case "true-false" : throw new AlreadyExists("This phone number : " + employeeDto.getPhoneNumber() + " is already in use");
```
Comment (vermurachi) [inline src/main/java/com/homework/homeworkWebApp/service/implementations/EmployeeServiceImpl.java]: Refactor this, no need to use switch here, an idea could be .ifPresent()...

**Comment by vermurachi** [comment]
Format code in all the files

File: src/main/java/com/homework/homeworkWebApp/model/Employee.java (hunk 2 above)
Comment (AlexandruBaris) [reply src/main/java/com/homework/homeworkWebApp/model/Employee.java]: removed

File: src/main/java/com/homework/homeworkWebApp/exceptions/AlreadyExists.java (hunk 1 above)
Comment (AlexandruBaris) [reply src/main/java/com/homework/homeworkWebApp/exceptions/AlreadyExists.java]: added suffix Exception

**Review by vermurachi — APPROVED** [review]

**Review by vermurachi — APPROVED** [review]

**Pull request merged**